src/testing/deploy/deploy_test.go

```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/pachyderm/pachyderm/v2/src/internal/testutil"
 )
 
+var valueOverrides map[string]string
```

Comment:
I'd add a comment that these are defined per-architecture via build tags (the file named _arm64_ is really just a build tag).  I will note that you can just write `func init() { switch runtime.GOARCH { case "arm64": valueOverrides = ... ; case "amd64": ... } }`, which I think would be clearer.

Reply:
ooo yes i like that. let me give it some more thought on the next PR, bob and I had come with this approach.